LayoutTests/svg/repaint/image-animation-with-zoom.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE HTML>
+<script src="../../fast/repaint/resources/text-based-repaint.js"></script>
+<style>
+html {
+    zoom: 2.2
+}
+* {
+    margin: 0;
+    padding: 0;
+}
+</style>
+<script>
+    window.testIsAsync = true;
+    window.onload = runRepaintTest;
+    function repaintTest() {
+        setTimeout(function() {
+            if (window.testRunner)
+            finishRepaintTest();
+        }, 220);
```

Comment:
Using `setTimeout(..., 220)` here is a little bit fragile. I do not think there I a guarantee that the CSS animation in the SVG image resource will finish in 200ms.

Reply:
Should I increase it? I looked into this test case from Chromium source and it is using 'rAf' but we don't have infrastructure in TestRunner to use "paintUnderInvalidationCheckingEnabled" and those bits so it wouldn't be possible to port latest Chromium test case code here.

Reply:
This is guaranteed to lead to flakey tests -- the timing cannot be guaranteed like this.